models/__tests__/supergoose.js

```diff
@@ -5,7 +5,7 @@
  */
 
 const mongoose = require('mongoose');
-const MongoMemoryServer = require('mongodb-memory-server').default;
+// const MongoMemoryServer = require('mongodb-memory-server').default;
```

Comment:
This is an npm package that needs to be installed, currently it's causing an error in your travis and tests.